scripts/util/pancancer_util.R

```diff
@@ -7,6 +7,8 @@
 #
 # Usage: sourced by scripts using function
 
+checkpoint::checkpoint("2017-06-01", checkpointLocation = ".")
```

Comment:
FYI, I'm now using conda to manage R dependencies without issues (assuming there are no packages not in conda). Something to consider for the future.